examples/player/simid_player.js

```diff
@@ -609,16 +609,69 @@ class SimidPlayer {
     console.log('The creative has asked for the player to ping ' + requestedUrlArray);
   }
 
+  /**
+   * Called when video playback is complete.
+   * @private
+   */
+  videoComplete() {
+    this.simidProtocol.sendMessage(MediaMessage.ENDED);
+
+    if (this.requestedDuration_ == NO_REQUESTED_DURATION) {
+      this.stopAd(StopCode.MEDIA_PLAYBACK_COMPLETE);
+    }
+
+    //If the request duration is longer than the ad duration, the ad extends for the requested amount of time
+    if (this.requestedDuration_ != UNLIMITED_DURATION) {
```

Comment:
Should this be an `else if`? (or a return inside the if block above?)

Otherwise, if the requestedduration_ == NO_REQUESTED_DUR, then well stop the ad, but also come down here and start a timer to stop the ad again in a few seconds

Reply:
yes, will change to else if